frontend/src/app/containers/HomePage/index.js

```diff
@@ -8,18 +8,7 @@ import "./index.scss";
 
 export default class HomePage extends React.Component {
   render() {
-    const {
-      hasAddon,
-      replaceState,
-      getCookie
-    } = this.props;
-
-    if (hasAddon === null) {
-      // If we are rendering the / page on the server, assume the addon is not installed.
-      return <HomePageNoAddon {...this.props} />;
-    }
-    if (getCookie("visit-count") === undefined && hasAddon) {
-      replaceState({}, "", "/experiments");
```

Comment:
IMHO we don't need to change the url to `/experiments` but it doesn't either way